code/drasil-lang/lib/Language/Drasil/Expr/Class.hs

```diff
@@ -174,13 +177,15 @@ class ExprC r where
   
   -- | Smart constructor for indexing.
   idx :: r -> r -> r
-  
+
+  idxOf :: r -> r -> r
```

Comment:
This needs a line of comment. I don't really know what it is / does / supposed to be.